tools/isolib/isolib.xml

```diff
@@ -21,7 +21,7 @@
         <requirement type="package" version="2.6">r-envipat</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
-        Rscript ${__tool_directory__}/isolib.R '${input_file}' '${adducts}' '${threshold}' '${append_adduct}' '${isotope_library}'
+        Rscript ${__tool_directory__}/isolib.R '${input_file}' '${ionization.adducts}' '${threshold}' '${append_adduct}' '${isotope_library}'
```

Comment:
The tool_dir needs to be single quotes.

Reply:
It also works like this - but ok will change the path to single quotes

Reply:
Put a space in path where you install your tools and it will stop working :-)